examples/private_set_intersection/private_set_intersection.sign

```diff
@@ -3,10 +3,10 @@ f41SClNtR4i46v2Tuh1fQLbt/ZqRr1lENajCW92jyP4=
 -----END PUBLIC KEY-----
 
 -----BEGIN SIGNATURE-----
-XTMXZqNPeVO5mfT+GURXMBXU3GrjVtIVlpdrnACbkgePpszy+RuL7wzrok6bU+P5
-1JY5gTU4vSU8kljTOzJvDw==
+z/qheXvmNUY/1YY06sw+1KQ0KjD9JF3/8aXHn9M8QtjTl0VY8rHWhnrWsWFiPmMG
+Ee7ZndTs3yVQRUIa7BMoBg==

```

Comment:
Out of curiosity, if you re-sign the same file with the same key, does it produce the same signature? Or does it use some nondeterministic value as part of the signature?